index.html

```diff
@@ -8,7 +8,10 @@
         <meta name="description" content="eXist Solutions Long Term Support (LTS) and Service Level Agreement (SLA) order form"/>
         <meta name="keywords" lang="en" content="eXistdb, nosql, database, xml, xquery, xforms, xsl, forms, web, development"/>
         <meta name="keywords" lang="en" content="eXistdb, nosql, database, xml, xquery, xforms, xsl, forms, web, entwicklung"/>
-        <meta name="author" content="eXist Solutions GmbH"/>
```

Comment:
I changed this to solutions since we all worked on the design and implementation of the website.